### PR TITLE
add sony yuga

### PR DESCRIPTION
**Pull request by jeroenqui**
My attempt to add support for sony yuga.
The only thing I wasn't sure about was the .adds and .removes in the manifest dir.
Please let me know if it needs more adjustments.

As this is the first 1920x1080 device for PAC, I guess it will need a custom bootanimation

### CODE REVIEW

**Comment by mpmckinney** [comment]
For the dependencies you're gonna want to have the account and revision for each of them. It uses revision and not branch.

**Comment by jeroenqui** [comment]
Ok, will edit that and commit the fixed files. Thanks.

File: dependencies/yuga.dependencies [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    "repository": "android_kernel_sony_apq8064",
+    "target_path": "kernel/sony/apq8064"
+    "revision": "cm-10.1"
+ },
+  {
+    "remote": "github",
+    "account": "CyanogenMod",
+    "repository": "android_packages_apps_FMRadio",
+    "target_path": "packages/apps/FmRadio",
+    "branch": "jellybean"
```
Comment (Nick0703) [inline dependencies/yuga.dependencies]: You need to change the "branch" to "revision"

**Comment by szlkiev** [comment]
@jeroenqui, added you as device maintainer, gave you access to vendor/pac.
Please commit device support all in just 1 commit

**Comment by jeroenqui** [comment]
Ok, will submit this again as just 1 commit.
Thanks for making me a device maintainer.

**Pull request closed without merging**